src/main/java/com/esri/samples/graphics_update/client_app/MoveGraphicsSample.java

```diff
@@ -0,0 +1,200 @@
+
+        // set the initial viewpoint of the map view
+        mapView.setViewpointCenterAsync(new Point(-286323, 7556611), 3000);
+    }
+
+    /**
+     * Method to update a graphic from a vehicle update message.  If the message has come from a new vehicle
+     * then a new graphic will be added.
+     * @param updateMessage
+     */
+    private void UpdateGraphic(UpdateMessage updateMessage) {
+        Point position = updateMessage.getPosition();
+
+        // does graphic already exist?
+        if (vehicles.containsKey(updateMessage.getVehicleID())) {
+            //update the existing graphic with a new point geometry
+            Graphic existingVehicle = vehicles.get(updateMessage.getVehicleID());
+            existingVehicle.setGeometry(updateMessage.getPosition());
+
+        } else {
+            // create new graphic
+            Graphic vehicleGraphic = new Graphic(position);
+            vehicleGraphic.getAttributes().put("Status", updateMessage.getStatus().toString());
+            graphicsOverlay.getGraphics().add(vehicleGraphic);
+
+            // add vehicle graphic to hash map
+            vehicles.put(updateMessage.getVehicleID(), vehicleGraphic);
+        }
+    }
+
+    /**
+     * Method to get symbols from style file and apply to renderer
+     * @param vehicleStyle
+     */
+
+    private void makeRenderer(SymbolStyle vehicleStyle) {
+
+        // create the unique value renderer
+        UniqueValueRenderer uniqueValueRenderer = new UniqueValueRenderer();
+        uniqueValueRenderer.getFieldNames().add("STATUS");
+
+        // default symbol
+        SimpleMarkerSymbol simpleMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFFFF0000,10);
+        uniqueValueRenderer.setDefaultSymbol(simpleMarkerSymbol);
+
+        // all of the potential status values for the vehicles
+        ArrayList<String> symbolNames = new ArrayList<>(Arrays.asList("Available", "Off duty", "On route", "Attending call"));
+
+        // loop through the symbols to create the unique values for each status
+        for (String symbolName : symbolNames) {
+            ListenableFuture<Symbol> searchResult = vehicleStyle.getSymbolAsync(Collections.singletonList(symbolName));
+            searchResult.addDoneListener(()-> {
+                try {
+                    Symbol symbol = searchResult.get();
+
+                    switch (symbolName) {
+                        case "Available" :
+                            UniqueValueRenderer.UniqueValue uniqueAvailableValue =
+                                    new UniqueValueRenderer.UniqueValue(symbolName, symbolName, symbol, Collections.singletonList("AVAILABLE"));
+                            uniqueValueRenderer.getUniqueValues().add(uniqueAvailableValue);
+                            break;
+                        case "Off duty" :
+                            List<Object> offDutyValue = new ArrayList<>();
+                            offDutyValue.add("OFF_DUTY");
```

Comment:
```suggestion
```
You can remove this since it's just being directly added as a string on line 159.